api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -320,6 +322,22 @@ message Pass {
   // The center frequency, in Hz, for uplinking in this pass. 0 if uplink is not available in
   // this pass.
   uint64 uplink_center_frequency_hz = 10;
+
+  // A mapping of channel set to its unique reservation token.
```

Comment:
I think this is the first ever mention of channels or channel set in the external API.

As such, we need to give a lot more information about what this is and how it should be used. For example, how does it related to the existing reservation token? What is a channel_set_id and where does it come from? etc.